src/main/java/com/codeit/sb01_deokhugam/domain/user/service/BasicUserService.java

```diff
@@ -98,8 +95,8 @@ public List<UserDto> findAllUsersIncludingDeleted() {
 	}
 
 	//유저 닉네임 변경
-	@Transactional
 	@Override
+	@Transactional(readOnly = false)
```

Comment:
`@Transactional`의 기본값은 false 입니다.  기본값이 있으니 속성을 따로 정의해주지 않아도 됩니다.
```suggestion
        @Transactional
```

Reply:
오호! 감사합니다